README.md

```diff
@@ -22,8 +22,9 @@
     - [Requirements](#db-requirements)
     - [Sync node](#sync-node-with-db)
 - [Examples](#examples)
-  - [1. Sync Osmosis on Kaon with DB-SYNC](#1-sync-osmosis-on-kaon-with-db-sync)
-  - [2. Sync Cosmos Hub on Mainnet with P2P-SYNC](#2-sync-cosmos-hub-on-mainnet-over-p2p-sync)
+  - [1. Sync Osmosis (Kaon) with DB-SYNC](#1-sync-osmosis-on-kaon-with-db-sync)
+  - [2. Sync Cosmos Hub (Mainnet) with P2P-SYNC](#2-sync-cosmos-hub-on-mainnet-over-p2p-sync)
+  - [2. Sync Cosmos Hub supervised](#3-sync-cosmos-hub-supervised)
```

Comment:
Do it as first example and get rid of "supervised", instead talk about auto-sync